src/config/dynamic-theme-fixes.config

```diff
@@ -5389,6 +5389,13 @@ CSS
 
 ================================
 
+community.mimecast.com/s/knowledge-hub
+
+INVERT
+.cb-section_background
```

Comment:
That totally inverts (make negative) the photo. Which is BAAAD!

I propose CSS fix instead of INVERT:
CSS
.cb-section_background {
    mix-blend-mode: multiply !important;
}

The picture will be dark but at least not inverted.